htdocs/compta/facture/card.php

```diff
@@ -2362,20 +2367,21 @@
 		$line = new FactureLigne($db);
 		$line->fetch(GETPOST('lineid', 'int'));
 		$percent = $line->get_prev_progress($object->id);
+		$progress = floatval(GETPOST('progress', 'int'));
```

Comment:
if progress is an int, you must use
$progress = (int) (GETPOST('progress', 'int'));
If it is a real, you must use
$progress = floatval(GETPOST('progress', 'alpha'));    // alpha to not sanitize the decimal separator